changelogs/fragments/1881-allow-s3_object-to-specify-content-type-in-metadata.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - s3_object - when doing a put and specifying "Content-Type" in metadata, this (since 6.0.0) erroneously set the Content-Type to None causing the put to fail. Fix now correctly honours the specified Content-Type (https://github.com/ansible-collections/amazon.aws/issues/1881).
```

Comment:
```suggestion
  - s3_object - when doing a put and specifying ``Content-Type`` in metadata, the module (since 6.0.0) erroneously set the `Content-Type`` to ``None`` causing the put to fail. Fix now correctly honours the specified ``Content-Type`` (https://github.com/ansible-collections/amazon.aws/issues/1881).
```